pylmod/__init__.py

```diff
@@ -20,7 +21,7 @@
         # not installed, but there is another version that *is*
         raise DistributionNotFound
 except DistributionNotFound:  # pragma: no cover
-    __version__ = 'Please install this project with setup.py'
+    __version__ = '0.1.0'  # hard coded value until we can debug RTD defect
```

Comment:
in `<project_root>/docs/conf.py` instead of importing, but yes, it is a constant so uppercase is appropriate